src/ui.form-date-picker-range/configs/default.config.js

```diff
@@ -52,10 +52,7 @@ export default /*tw*/ {
   periodDateWeekList: "",
   periodDateYearList: "grid grid-cols-3 grid-rows-1 gap-0.5",
   periodDateQuarterList: "",
-  periodDate: `
-    cursor-pointer block w-full rounded-lg py-3 text-center text-sm font-medium text-gray-900 hover:rounded-lg hover:bg-brand-50
-    disabled:opacity-50 disabled:cursor-not-allowed
-  `,
+  periodDate: "w-full hover:!bg-brand-100 disabled:cursor-not-allowed",
```

Comment:
use default hover `hover:!bg-brand-100` 
`disabled:cursor-not-allowed` - do we really need this class? I think it should be removed.